pyproject.toml

```diff
@@ -42,6 +42,10 @@ langchain = "^0.0.198"
 google-cloud-aiplatform = {version = "^1.26.0", optional = true}
 miniaudio = "^1.59"
 boto3 = "^1.28.28"
+google-api-python-client = "^2.125.0"
```

Comment:
i don't want these I just want `dateutils`. Somehow that came bundled with these or something? @ericmei send help pls